FreeRTOS/Demo/CORTEX_A5_SAMA5D3x_Xplained_IAR/AtmelFiles/usb/device/core/USBDDriver.c

```diff
@@ -562,7 +562,7 @@ void USBDDriver_Initialize(
 
     if (pInterfaces != 0) {
 
-        memset(pInterfaces, sizeof(pInterfaces), 0);
+        memset(pInterfaces, 0, sizeof(uint8_t));

```

Comment:
What is the purpose of this statement I wonder.
Because this is not setting the whole array to 0, it is just setting the first element to 0. Does that mean that we are using entry with a `0` as a delimiter?

Just trying to understand why is this being done at all :)

Reply:
IMHO, "multiple USB driver" is not supported in this version. I believe this is for future use. Confirming with partner but no response yet.